source/jormungandr/tests/bragi_autocomplete_tests.py

```diff
@@ -1615,3 +1619,29 @@ class TestDistributedAutocompleteAndRouting(
     AbstractAutocompleteAndRouting, NewDefaultScenarioAbstractTestFixture
 ):
     pass
+
+
+@dataset({'main_routing_test': MOCKED_INSTANCE_POI_DATASET_CONF}, global_config={'activate_bragi': True})
+class AbstractAutocompletePoiDataset:
+    def test_poi_dataset(self):
+
+        """
+        Basic test for autocompletion with poi_dataset.
+        We setup the mock instance to have a poi_dataset 'priv.bob', and then build a query for bragi.
+        We verify that the query to bragi correctly adds a poi_dataset[] parameter.
+        """
+
+        with requests_mock.Mocker() as m:
+            m.get('https://host_of_bragi/autocomplete', json={})
+            self.query_region('places?q=bob&from=3.25;49.84')
+            # m.get(url, json={})
+            # self.query_region('places?q=bob')
+            assert m.called
+            params = m.request_history[0].qs
+            assert params
+            assert params.get('poi_dataset[]') == ['priv.bob']
+
+
```

Comment:
you need to add the following tests:
  - places on a coverage
  - reverse
  - journeys

Reply:
:+1: